learning/tour-of-beam/learning-content/IO/text-io/text-io-gcs-write/description.md

```diff
@@ -80,4 +80,8 @@ It is important to note that in order to interact with **GCS** you will need to
 ```
 --tempLocation=gs://my-bucket/temp
 ```
-{{end}}
\ No newline at end of file
+{{end}}
+
+### Playground exercise
+
+You can write PCollection to a file. Use DoFn to generate numbers and write them down by filtering.
```

Comment:
Please follow the pattern e.g. first give a small description of what the playground example is doing and then give a challenge

Reply:
done